rudder-reports/debian/copyright

```diff
@@ -1,24 +1,15 @@
-This package was debianized by root <root@rudderVM.normation.com> on
-Mon, 17 Jan 2011 08:19:10 -0500.
+This package was debianized by:
 
-It was downloaded from <url://example.com>
+    Jonathan Clarke <Jonathan Clarke <jonathan.clarke@normation.com>> on Tue, 06 Oct 2009 01:20:26 +0200
 
 Upstream Author(s):
 
-    <put author's name and email here>
-    <likewise for another author>
+    Normation - http://www.normation.com
 
 Copyright:
 
-    <Copyright (C) YYYY Name OfAuthor>
-    <likewise for another author>
+    Copyright (C) 2011 Normation SAS
```

Comment:
This copyright file was not configured, unsure about what to put here, and if it needs a separate ticket.